azure-pipelines.yml

```diff
@@ -42,3 +41,39 @@ jobs:
     displayName: Docker setup and build
 
   - publish: $(Build.ArtifactStagingDirectory)
+
+- job: Windows_Build
+  variables:
+    REPO_NAME: $(Build.Repository.Name)
+    REPO_BRANCH: $(System.PullRequest.SourceBranch)
+    PR_NUMBER: $(System.PullRequest.PullRequestId)
+    ARTIFACT_DIR: $(Build.ArtifactStagingDirectory)
+    COMPILER: msvc
+    QTDIR: 'C:\Qt\5.14.1\msvc2017_64'
```

Comment:
Hmm, does Azure let us reference variables? So like could we do
```
QTVER: 5.14.1
QTDIR: 'C:\Qt\$(QTVER)\msvc2017_64'
```
That would be pretty sweet, and work well with the Qt thing I mentioned earlier.